packages/api-client/src/connections/EntityApi.ts

```diff
@@ -37,7 +35,7 @@ const recurseFilter = (
     const value = filter.comparisonValue;
     filterArray.push([
       nestedKeys,
-      // @ts-ignore
```

Comment:
`@ts-ignore` gives a linting error, but `ts-expect-error` doesn't as long as we provide and explanation. So we should always use that 👍